src/Polly.Extensions/DependencyInjection/ConfigureResiliencePipelineRegistryContext.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Polly.DependencyInjection;
```

Comment:
Delete this file?

Reply:
Apologies -- this appears to have been 'resurected' when I synced the changes -- I had removed the file :-/